packages/side-nav/src/SideNavItem/SideNavItem.tsx

```diff
@@ -171,28 +176,17 @@ const SideNavItem: ExtendableBox<
         ref={forwardedRef}
         onClick={onClick}
       >
-        {accessibleGlyph && (
-          <span className={glyphWrapper}>
-            {accessibleGlyph}
-
-            <CollapsedSideNavItem active={active}>
-              {accessibleGlyph}
-            </CollapsedSideNavItem>
-          </span>
-        )}
+        <AccessibleGlyph
+          isActiveGroup={active}
+          accessibleGlyph={accessibleGlyph}
+          className={glyphWrapperStyle}
+        />
 
         {renderedChildren}
       </Box>
 
       {hasNestedItems && (
-        <ul
-          className={css`
-            list-style: none;
-            padding-inline-start: 0;
-          `}
-        >
-          {renderedNestedItems}
-        </ul>
+        <ul className={nestedULStyle}>{renderedNestedItems}</ul>
```

Comment:
nit - I personally. feel like it should be `nestedUlStyle` even if the `l` kind of looks like an `I`

Reply:
I agree, i'll change this.